custom/icds_reports/tasks.py

```diff
@@ -1820,8 +1820,8 @@ def get_data_not_in_ucr(status_record):
         for doc_id, doc_subtype, sql_modified_on in chunk:
             if doc_id in doc_id_and_inserted_in_ucr:
                 # This is to handle the cases which are outdated. This condition also handles the time drift of 1 sec
-                # between main db and ucr db. i.e  doc will even be included when inserted_at-sql_modified_on <= 1 sec
-                if sql_modified_on - doc_id_and_inserted_in_ucr[doc_id] >= timedelta(seconds=-1):
+                # between main db and ucr db. i.e  doc will even be included when inserted_at-sql_modified_on < 2 sec
```

Comment:
@rohit-dimagi what prompted this change? Did you have evidence of a lag of more than 2s?